src/core/trtserver.cc

```diff
@@ -119,31 +119,58 @@ TrtServerError::TrtServerError(
 //
 class TrtServerSharedMemoryBlock {
  public:
+#ifdef TRTIS_ENABLE_GPU
```

Comment:
Have 2 constructors, one for system and one for cuda.

Reply:
Sure